modules/axp/Animal/lib/Cow.pm

```diff
@@ -131,4 +136,6 @@ EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 =cut
 
+__PACKAGE__->meta->make_immutable( inline_constructor => 0 );
```

Comment:
Apparently we have to do this to avoid warnings, since we've got a `before` hook on the `new` constructor.